tests/tt_metal/tt_metal/unit_tests_common/common/dprint_fixture.hpp

```diff
@@ -93,6 +93,19 @@ class DPrintFixture: public ::testing::Test {
             );
             return true;
         }
+
+        // Also skip all devices after device 0 for grayskull. This to match all other tests
+        // targetting device 0 by default (and to not cause issues with BMs that have E300s).
+        // TODO: when we can detect only supported devices, this check can be removed.
+        if (this->arch_ == tt::ARCH::GRAYSKULL && device_id > 0) {
```

Comment:
can you file an issue for detecting only supported devices? thx :)

Reply:
https://github.com/tenstorrent-metal/tt-metal/issues/4569

Filed an issue here, not sure what tags need to be added or which board to add it to though.

Reply:
thanks